lib/le.rb

```diff
@@ -44,22 +42,16 @@ def self.new(token, region, options={})
     logger
   end
 
-  def self.checkParams(token, region, opt_datahub_enabled, opt_udp_port)
-    # Check if the key is valid UUID format
+  def self.check_params(token, region, opt_datahub_enabled, opt_udp_port)
+    # test Token only when DataHub and UDP are not enabled
+    return unless !opt_datahub_enabled && !opt_udp_port
 
-    if (!opt_datahub_enabled && !opt_udp_port)  # test Token only when DataHub and UDP are not enabled
```

Comment:
Are we dropping Datahub support?

Reply:
No, check the line above, just refactoring, before this function only contained an `if`.
Nothing executed if it didnt trigger, now its `return if not <DATAHUB_CHECK>` and then the code within the `if`.